src/tailwind.scroll-shadows.js

```diff
@@ -0,0 +1,62 @@
+import plugin from 'tailwindcss/plugin.js';
+
+/**
```

Comment:
Almost everything here is taken directly from [the SASS implementation](https://github.com/hypothesis/frontend-shared/blob/d5181845da9b4e5985ac90e09c25f86cdcc7ac12/styles/mixins/patterns/_containers.scss#L154-L229), but restructured for Tailwind.